### PR TITLE
[ci] Platform Switch Option

### PR DESCRIPTION
**Pull request by ppenna**
# Description

This PR closes https://github.com/microsoft/demikernel/issues/1316

### CODE REVIEW

File: tools/ci/config/test/linux/catnapw.yaml
Comment (anandbonde) [inline tools/ci/config/test/linux/catnapw.yaml:1]: Is this file needed anymore?

File: tools/ci/config/test/windows/catnap.yaml
Comment (anandbonde) [inline tools/ci/config/test/windows/catnap.yaml:1]: This file is identical to the `linux/catnap.yaml` file. Do you foresee these 2 files diverging in future? Also, if we run the same commands for linux/windows, why do we need platform distinction at all? Under what circumstances do the CI runs for the 2 platforms differ? (I know that paths may be 1 difference, what are the others?)

File: tools/demikernel_ci.py [hunk 1]
```diff
@@ -71,7 +72,7 @@ def run_pipeline(
     # STEP 4: Run integration tests.
     if test_integration:
         if status["checkout"] and status["compile"]:
-            if libos == "catnap" or libos == "catnapw" or libos == "catloop" or libos == "catnip" or libos == "catpowder":
+            if libos == "catnap" or libos == "catloop" or libos == "catnip" or libos == "catpowder":
```
Comment (anandbonde) [inline tools/demikernel_ci.py:75]: How about using the `libos in [...]` syntax here?

**Review by anandbonde — APPROVED** [review]
Looks good.

File: tools/ci/config/test/linux/catnapw.yaml
Comment (ppenna) [reply tools/ci/config/test/linux/catnapw.yaml:1]: No it is not. Thanks for catching this.

File: tools/ci/config/test/windows/catnap.yaml
Comment (ppenna) [reply tools/ci/config/test/windows/catnap.yaml:1]: Thanks fot cathing this. I fixed it so it is a symlink instead.

File: tools/demikernel_ci.py (hunk 1 above)
Comment (ppenna) [reply tools/demikernel_ci.py:75]: Oh yeah!
